specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/CommonDefinitions.json

```diff
@@ -2923,6 +2923,14 @@
               "$ref": "#/definitions/DaprConfig",
               "description": "Dapr configuration of the app."
             },
+            "workloadProfileName": {
```

Comment:
Are there example updates?

Reply:
This PR is in release branch and was looking into few of the previous PRs which do not have the examples updates for properties used on a specific scenario. These properties are applicable for one of the scenarios in this Site schema (Site have multiple scenarios of Web, functions, logic app, etc), so may not be also accurate to update.

Reply:
Since these properties are for external customers and these examples are feed into public rest api doc that customer follows, it is always good to add examples for all the properties, so that Cx can get idea how the property values look like. From your explanation, it is not clear what is the concern here adding/updating the examples for the new properties added in this PR.
I would suggest consider adding them in your examples.